.github/workflows/release.yml

```diff
@@ -13,12 +13,15 @@ jobs:
     steps:
     - name: git co
       uses: actions/checkout@v1
-
+    - name: Get Tag Info
+      id: release-tag-meta
+      uses: ./.github/actions/release-tag-meta
+      with:
+        git-ref: ${{ github.ref }}
     - name: make package
       env:
         CARGO_RELEASE: 1
-      run: PACKAGE_VERSION=$(echo $GITHUB_REF | sed -E 's,^refs/tags/release/,,') make package
-
+      run: PACKAGE_VERSION=${{steps.release-tag-meta.name}} make package
```

Comment:
how about:
```yaml
run: PACKAGE_VERSION=$(echo "$GITHUB_REF" | sed s,^refs/tags/release/,,) make package
```
...and then forego `Dockerfile`, `entrypoint.sh`, and `action.yml `? i could see utility in the custom action if it was doing more stuff.